ktor-io/common/test/ReadUtf8LineTest.kt

```diff
@@ -57,4 +80,154 @@ class ReadUtf8LineTest {
         channel.readUTF8LineTo(buffer, 1024)
         assertEquals("2", buffer.toString())
     }
+
+    @Test
+    fun `test reading large amount of lines completes in a reasonable time`() = runTest(timeout = 5.seconds) {
+        var count = 0
+        val numberOfLines = 200_000
+        val channel = writer {
+            repeat(numberOfLines) { i ->
+                channel.writeStringUtf8("line $i\n")
+            }
+        }.channel
+
+        val out = StringBuilder()
+        while (channel.readUTF8LineTo(out) && count < numberOfLines) {
+            count++
+        }
+
+        assertEquals(numberOfLines, count)
+        assertEquals(2_088_890, out.length)
+    }
+
+    @Test
+    fun `test reading long lines completes in a reasonable time`() = runTest(timeout = 5.seconds) {
```

Comment:
This test times out. It seems too low timeout for CI agent.
Probably we should remove these two tests and just run io-benchmarks 🤔